packages/api-tests/test/window.spec.js

```diff
@@ -1038,4 +1041,252 @@ describe('Window API', function(done) {
       return chainPromises(steps);
     });
   });
+
+  describe('Event Listeners', function() {
+    const addListener = (event) => {
+      /* eslint-disable no-undef */
+      const script = (event, callback) => {
+        const currentWindow = ssf.Window.getCurrentWindow();
+        callback(currentWindow.addListener(event, () => { window.listenEventResult = true; }));
+      };
+      /* eslint-enable no-undef */
+      return executeAsyncJavascript(app.client, script, event);
+    };
+
+    const retrieveListenerResult = () => {
+      /* eslint-disable no-undef */
+      const script = (callback) => {
+        callback(window.listenEventResult);
+      };
+      /* eslint-enable no-undef */
+      return executeAsyncJavascript(app.client, script);
+    };
+
+    const retrieveDelay = 500;
+
+    it.skip('Should emit the blur event #ssf.WindowEvent.blur', function() {
+      // Electron works but OpenFin doesnt seem to set the value
```

Comment:
If this is a difference in how the containers work then I think we should add it as an issue.